warp10/src/main/thrift/io_warp10_continuum_store_thrift_data.thrift

```diff
@@ -95,6 +96,11 @@ struct KafkaDataMessage {
    * Minimum age of cells to delete (in ms)
    */
   8: optional i64 deletionMinAge,
+
+  /**
+   * Optional write token to log metrics in deletion requests
+   */
+  9: optional io_warp10_quasar_token_thrift_data.WriteToken writeToken,
```

Comment:
WDYM with "add a field"? A new property in Metadata (seems a heavy change) or using the "attributes" map?